docs/zkapps/advanced-snarkyjs/actions-and-reducer.mdx

```diff
@@ -54,21 +54,21 @@ let stateType = Bool;
 // example actions data
 let actions = [[Field(1000)], [Field(2)], [Field(100)]];
 
-// state and actionsHash before applying actions
+// state and actionState before applying actions
 let initial = {
   state: Bool(false),
-  actionsHash: Reducer.initialActionsHash,
+  actionState: Reducer.initialActionState,
 };
 
-let { state, actionsHash } = this.reducer.reduce(
+let { state, actionState } = this.reducer.reduce(
   actions,
   stateType,
   (state: Bool, action: Field) => state.or(action.equals(1000)),
   initial
 );
 ```
 
-What we called `acc` above is now called `state`; we also have to pass in the state's type as a parameter. In addition, we have to pass in an `actionsHash` which refers to one particular point in the actions history. Like `Array.reduce`, `Reducer.reduce` takes a callback which has the signature `(state: S, action: A) => S`, where `S` is the `stateType` and `A` is the `actionType`. It returns the result of applying all the actions, in order, to the initial `state`. In this example, the returned `state` is `Bool(true)` because one of the actions in the list is `Field(1000)`. Reduce also returns the new actionsHash -- so you can store it for using it when you reduce the next batch of actions. One last difference to JS reduce is that this takes a _list of lists_ of actions instead of a flat list. Each of the sublists are the actions that were dispatched in one account update (e.g., while running one smart contract method).
+What we called `acc` above is now called `state`; we also have to pass in the state's type as a parameter. In addition, we have to pass in an `actionState` which refers to one particular point in the actions history. Like `Array.reduce`, `Reducer.reduce` takes a callback which has the signature `(state: S, action: A) => S`, where `S` is the `stateType` and `A` is the `actionType`. It returns the result of applying all the actions, in order, to the initial `state`. In this example, the returned `state` is `Bool(true)` because one of the actions in the list is `Field(1000)`. Reduce also returns the new actionState -- so you can store it for using it when you reduce the next batch of actions. One last difference to JS reduce is that this takes a _list of lists_ of actions instead of a flat list. Each of the sublists are the actions that were dispatched in one account update (e.g., while running one smart contract method).
```

Comment:
```suggestion
The `acc` shown earlier is now `state`; you also have to pass in the state's type as a parameter. In addition, you have to pass in an `actionState` which refers to one particular point in the action's history. Like `Array.reduce`, `Reducer.reduce` takes a callback that has the signature `(state: S, action: A) => S`, where `S` is the `stateType` and `A` is the `actionType`. It returns the result of applying all the actions, in order, to the initial `state`. In this example, the returned `state` is `Bool(true)` because one of the actions in the list is `Field(1000)`. Reduce also returns the new actionState -- so you can store it to use when you reduce the next batch of actions. One last difference to JS reduce is that this takes a _list of lists_ of actions instead of a flat list. Each of the sublists are the actions that were dispatched in one account update (for example, while running one smart contract method).
```

Reply:
@MartinMinkov  let's use [present tense](https://developers.google.com/style/tense) and [timelessness](https://developers.google.com/style/timeless-documentation)

I want to learn more about how we can describe the feature without bounding with "now"